tools/gluten-it/common/src/main/java/io/glutenproject/integration/tpc/command/QueriesCompare.java

```diff
@@ -41,10 +41,13 @@ public class QueriesCompare implements Callable<Integer> {
   @CommandLine.Option(names = {"--iterations"}, description = "How many iterations to run", defaultValue = "1")
   private int iterations;
 
+  @CommandLine.Option(names = {"--verify-spark-plan"}, description = "Verify whether spark plan changes", defaultValue = "true")
```

Comment:
Can we disable it by default ? it's possiable there are other tests except tpch and tpcds